src/ralph/dhcp/views.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from functools import partial
+
+from django.db.models import Q
+from django.http import (
+    HttpResponse,
+    HttpResponseBadRequest,
+    HttpResponseNotFound,
+    HttpResponseNotModified
+)
+from django.utils.http import http_date, parse_http_date_safe
+from django.views.generic.base import TemplateView, View
+
+from ralph.assets.models.components import Ethernet
+from ralph.data_center.models.physical import DataCenter
+from ralph.dhcp.models import DHCPEntry, DHCPServer
+from ralph.networks.models.networks import Network, NetworkEnvironment
+
+logger = logging.getLogger(__name__)
+
+
+class LastModifiedMixin(object):
+    """Add last modified to HTTP response if last_modified attr is exist."""
+
+    @classmethod
+    def last(cls, qs, last_items=None, filter_dict={}):
```

Comment:
filter_dict={}? maybe better is filter_dict=None, and if filter_dict is None ....